com.coherentsolutions.training.auto.web.pashkovskaya/src/main/java/pages/HomePage.java

```diff
@@ -0,0 +1,25 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class HomePage extends BasePage{
+    public HomePage(WebDriver driver) {
+        super(driver);
+    }
+    @FindBy(xpath = "//span[@class=\"user-account__name\"]") public WebElement username;
```

Comment:
let's place @FindBy annotation and WebElement declaration on two separate lines
it will be more readable